flow/embedded_views.h

```diff
@@ -21,7 +21,14 @@
 namespace flutter {
 
 // TODO(chinmaygarde): Make these enum names match the style guide.
-enum MutatorType { clip_rect, clip_rrect, clip_path, transform, opacity };
+enum MutatorType {
+  clip_rect,
+  clip_rrect,
+  clip_path,
+  transform,
+  opacity,
+  backdrop_filter
```

Comment:
nit: while you're here, can you just resolve the TODO above? https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
Done